pymc/math.py

```diff
@@ -341,6 +334,11 @@ def log1mexp_numpy(x, *, negative_input=False):
     For details, see
     https://cran.r-project.org/web/packages/Rmpfr/vignettes/log1mexp-note.pdf
     """
+    warnings.warn(
+        "pymc.math.log1mexp_numpy is being deprecated. Use log1mexp instead.",
```

Comment:
Also need to implement it locally here, so that it can be safely removed later:

https://github.com/pymc-devs/pymc/blob/f302882da263d8569af1f55b3d758c614ecb3799/tests/distributions/test_continuous.py#L427

Reply:
OK, will do, I'll check for all cases where those numpy functions show up, is that just to have the comment visually written there so it can be identified from looking at the file?

Reply:
Sorry, what do you mean?

Reply:
I mean no other part of the codebase should be using a function marked for deprecation (other than code that is testing that function directly). That ensures then when the time comes all we need to do is remove the deprecated code.